### PR TITLE
Added Desktop_Version

### PR DESCRIPTION
**Pull request by RaviTejaM9602**
> Created Template of Desktop Version 
> Used Media queries to have the flexibility in viewing for both Mobile and Desktop versions.
> I used major properties Grid and Flex to design both Versions.
> Added padding for left and right for work-section not to collide with its borders.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,14 +49,16 @@ <h5>LET'S CONNECT</h5>
           </ul>     
       </section>  
    </section>
-
+  </main>
+  
       <!--Work Section -1  -->
    
-      <section class="work-section">
+  <section class="work-section">
```
Comment (petudeveloper) [inline index.html:56]: Please try to follow the given Figma design. Though your design is not bad, I think it is better to follow the given desktop work-section design. 

Your Desktop version doesn't look like the one in the design you chose between the sizes 992px and 1100px. Please try adding the corresponding CSS property to stick to the design as much as possible (e.g. margins, padding, etc), 

Check the images below to have a better understanding. 
**Figma design:** 
Capture2
**Your design:** 
Capture

**Review by petudeveloper — CHANGES_REQUESTED** [review]
Hi @RaviTejaM9602 , 

Good job so far! 
There are some issues that you still need to work on to go to the next project but you are almost there! :muscle:

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -74,16 +81,18 @@ <h4>Canopy</h4>
...
 
         <!--Work Section 2-->
 
         <div class="work-container">
           <div class="img img-2">
-            <img src="./images/Snapshoot2.png" alt="second-work">
-            <h2>Multi-Post</h2>
+            <img src="./images/Desktop/Grid-2/Snapshoot_Portfolio_2.svg.svg" alt="second-work">
+            <div class="content">
+              <h2>Multi-Post</h2>
```
Comment (rotimiazeez) [inline index.html:95]: - [ ] As mentioned by the previous reviewer, Kindly make sure your design matches that of your preferred template😉
- Your cards are not following the design on the Figma template🙄, I'd suggest you use 👇
```
work-container:nth-child(even) .cards {
    flex-direction: row-reverse !important;
}
```
to reverse the card content and image as seen in the original design🤔
- Figma design
screenshot (53)
- Your design
screenshot (52)

Comment (rotimiazeez) [reply index.html:95]: - [ ] Kindly fix the broken image on your card too for a better user interface👌

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @RaviTejaM9602 👋, 

### Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.😌

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (RaviTejaM9602) [reply index.html]: I accidentally missed the image link....... Now I added it

**Review by Cathella — APPROVED** [review]
## 🟢 STATUS: APPROVED 🎉

### Hi team 👋🏻, 

You had done a tremendous job on this design 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] [OPTIONAL] I kindly request that you add more padding to the "about myself" text. More white space allows the eyes of the website visitor to quickly identify the content on it!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**